src/enable_variable_ops/ngraph_assign_op.cc

```diff
@@ -48,9 +48,10 @@ namespace ngraph_bridge {
 // Computes *input[0] = input[1]
 class NGraphAssignOp : public OpKernel {
  private:
-  bool just_looking_;
+  bool is_tf_modifying_;
   bool copy_to_tf_;
   int ng_graph_id_;
+  bool just_looking_;
```

Comment:
Should I change the variable name from just_looking to something else? May be is_not_changing or is_tracked or will_not_be_modified or ?

Reply:
Yes - please do so.

Reply:
is_not_modified, is_not_modified_by_tf

Reply:
or just_looking, tf_just_looking

Reply:
Can you please add a comment in the file here what `just_looking_` and `is_tf_modifying_` would mean, and why we need 2 flags instead of just one (normal path vs var-opt path etc)

Reply:
`flib_graph_2.pbtxt` might need changes (adding `is_tf_modifying_` attribute): https://github.com/tensorflow/ngraph-bridge/blob/master/test/python/flib_graph_2.pbtxt